src/Catalog/Helpers/FeedHelpers.cs

```diff
@@ -68,15 +76,64 @@ public static HttpClient CreateHttpClient(bool verbose)
             return null;
         }
 
-        public static async Task<SortedList<DateTime, IList<PackageDetails>>> GetPackages(HttpClient client, Uri uri, string keyDateProperty)
+        /// <summary>
+        /// Builds a <see cref="Uri"/> for accessing the metadata of a specific package on the feed.
+        /// </summary>
+        public static Uri MakeUriForPackage(string source, string id, string version)
+        {
+            var uri = new Uri(string.Format($"{source.Trim('/')}/Packages(Id='{id}',Version='{version}')"));
```

Comment:
Nice catch!

I based this off another line that had the extra `string.Format` and just must not have noticed it.